src/lib/times.js

```diff
@@ -22,33 +22,66 @@ if (process.env.NODE_ENV === 'development') {
   times.unshift({ id: 'debug', icon: 'nightly.svg', title: browser.i18n.getMessage('timeRealSoonNow')});
 }
 
+
+const i18n_formats = ((locale) => {
+  const formats = {
+    default: {
```

Comment:
Might be worth splitting these out into a separate JSON file, [which we can import like a JS module](https://github.com/bwinton/SnoozeTabs/blob/master/src/lib/metrics.js#L7). Thinking that could make it easier for localizers to find and add entries.